tests/test_KafkaError.py

```diff
@@ -56,3 +56,9 @@ def a_method(self):
     err = MyExc()
     assert err.a_method() == "yes"
     assert isinstance(err, KafkaException)
+
+
+def test_kafkaError_custom_msg():
+    err = KafkaError(KafkaError._ALL_BROKERS_DOWN, "Mayday!")
+    assert err == KafkaError._ALL_BROKERS_DOWN
+    assert err.str() == "Mayday!"
```

Comment:
Add a KafkaException-wrapped KafkaError test too, and one where you specify an unknown code, e.g., 12345.